src/procedures/proc_fulltext_create_index.c

```diff
@@ -66,11 +70,19 @@ static ProcedureResult _validateIndexConfigMap
 		}
 	}
 
+	//--------------------------------------------------------------------------
+	// validate language
+	//--------------------------------------------------------------------------
+
 	if(lang_exists) {
 		if(SI_TYPE(lang) != T_STRING) {
 			ErrorCtx_SetError("Language must be string");
 			return PROCEDURE_ERR;
 		}
+		if(RSLanguage_Find(lang.stringval) == RS_LANG_UNSUPPORTED) {
```

Comment:
`RSLanguage_Find` should be part of the `RediSearch Low Level API`, let's wait for the search team to introduce it